doc/whats_new/v1.6.rst

```diff
@@ -81,13 +81,20 @@ Changelog
   whether a given estimator is of category clusterer.
   :pr:`28936` by :user:`Christian Veenhuis <ChVeen>`.
 
+:mod:`sklearn.cluster`
+......................
+
+- |API| The `copy` parameter of :class:`cluster.Birch` was deprecated in 1.6 and will be
+  removed in 1.8. It has no effect as a copy is always made.
```

Comment:
The reason is not that a copy is always made but that this estimator doesn't perform in place operations on the input data.